pkg/ext/stores/tokens/tokens.go

```diff
@@ -662,41 +718,46 @@ func (t *SystemStore) update(sessionID string, fullPermission bool, token *ext.T
 		if apierrors.IsNotFound(err) {
 			return nil, apierrors.NewNotFound(GVR.GroupResource(), token.Name)
 		}
-		return nil, apierrors.NewInternalError(fmt.Errorf("failed to retrieve token %s: %w",
-			token.Name, err))
+		return nil, apierrors.NewInternalError(fmt.Errorf("failed to retrieve token: %w", err))
 	}
 	currentToken, err := tokenFromSecret(currentSecret)
 	if err != nil {
-		return nil, apierrors.NewInternalError(fmt.Errorf("failed to extract token %s: %w",
-			token.Name, err))
+		return nil, apierrors.NewInternalError(fmt.Errorf("failed to extract token: %w", err))
 	}
 
 	if token.ObjectMeta.UID != currentToken.ObjectMeta.UID {
-		return nil, apierrors.NewBadRequest(fmt.Sprintf("rejecting change of token %s: forbidden to edit kubernetes UID",
-			token.Name))
+		return nil, apierrors.NewBadRequest("forbidden to edit kubernetes UID")
```

Comment:
```suggestion
		return nil, apierrors.NewBadRequest("forbidden to edit UID")
```

Reply:
Here and everywhere else "forbidden to edit" implies the lack of permissions, whereas in fact it's an immutable field. No matter the level of permissions the value should not change once created.
I'd suggest to use an error message "UID is immutable".

Reply:
good point

Reply:
done. commit 6434fb02ab93bbf500610c0d6214ca9daab4515e